### PR TITLE
added blue button in home page with container

### PR DESCRIPTION
**Pull request by yannklein**
image

What do you think @Donald-Chow do you like the color?

### CODE REVIEW

**Comment by Donald-Chow** [comment]
> image
> What do you think @Donald-Chow do you like the color?

We should use `red` to match Le Wagon color theme.

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,2 +1,6 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+<div class="container  my-5">
+  <h1>Pages#home</h1>
+  <p>Find me in app/views/pages/home.html.erb</p>
+
```
Comment (yannklein) [inline app/views/pages/home.html.erb:4]: No need for a space here

**Review by yannklein — COMMENTED** [review]
Check those out!

**Comment by yannklein** [comment]
@Donald-Chow wha' you think?

**Comment by Donald-Chow** [comment]
> @Donald-Chow wha' you think?

LGTM

**Pull request merged**